lib/apipie/generator/generator.rb

```diff
@@ -0,0 +1,2 @@
+module Apipie::Generator
+end
```

Comment:
these small files only defining the module is good for auto-loading in Rails, but does not add much value in a gems where everything is explicitly `require`d 

it does not hurt either, except for adding extra files and bytes to the project.
I'm kinda leaning toward removing... you got another opinion?

Reply:
I used them in order for the `::` notation to work. 

For example 
```ruby 
class Apipie::Generator::Swagger::Type
end
```

instead of 
```ruby
module Apipie
  module Generator
    module Swagger
      class Type
      end
    end
  end
end
```

I'm fine with either way

Reply:
right, that make sense... 💯  lets keep them instead adding the indent 👍🏽